### PR TITLE
Initial and terminal copresheaves

### PR DESCRIPTION
**Pull request by FernandoChu**
Adding initial and terminal copresheaves.

### CODE REVIEW

File: src/category-theory/copresheaf-categories.lagda.md [hunk 1]
```diff
@@ -307,6 +315,108 @@ module _
...
+    hom-initial-copresheaf-Precategory X ＝ τ
+  is-unique-hom-initial-copresheaf-Precategory X τ =
+    eq-htpy-hom-family-natural-transformation-Precategory
+      ( C)
+      ( Set-Precategory l)
+      ( obj-initial-copresheaf-Precategory)
+      ( X)
+      ( hom-initial-copresheaf-Precategory X)
+      ( τ)
+      ( λ x → eq-htpy λ {(map-raise ())})
```
Comment (fredrik-bakke) [inline src/category-theory/copresheaf-categories.lagda.md:356]:
```suggestion
      ( λ x → eq-htpy (λ where (map-raise ())))
```

File: src/category-theory/copresheaf-categories.lagda.md [hunk 2]
```diff
@@ -307,6 +315,108 @@ module _
...
+    hom-terminal-copresheaf-Precategory X ＝ τ
+  is-unique-hom-terminal-copresheaf-Precategory X τ =
+    eq-htpy-hom-family-natural-transformation-Precategory
+      ( C)
+      ( Set-Precategory l)
+      ( X)
+      ( obj-terminal-copresheaf-Precategory)
+      ( hom-terminal-copresheaf-Precategory X)
+      ( τ)
+      ( λ x → eq-htpy λ _ → eq-is-prop is-prop-raise-unit)
```
Comment (fredrik-bakke) [inline src/category-theory/copresheaf-categories.lagda.md:407]:
```suggestion
      ( λ x → eq-htpy (λ _ → eq-is-prop is-prop-raise-unit))
```

File: src/category-theory/copresheaf-categories.lagda.md [hunk 3]
```diff
@@ -307,6 +315,108 @@ module _
...
+    obj-initial-copresheaf-Precategory
+  pr1 (pr2 initial-copresheaf-Precategory X) =
+    hom-initial-copresheaf-Precategory X
+  pr2 (pr2 initial-copresheaf-Precategory X) τ =
+    is-unique-hom-initial-copresheaf-Precategory X τ
+```
+
+### The terminal copresheaf
+
+```agda
```
Comment (fredrik-bakke) [inline src/category-theory/copresheaf-categories.lagda.md:371]: Can you add just a sentence stating that the terminal copresheaf is the constant functor at the terminal set?

File: src/category-theory/copresheaf-categories.lagda.md [hunk 4]
```diff
@@ -307,6 +315,108 @@ module _
             ( pr2 w)))
 ```
 
+### The initial copresheaf
+
+```agda
```
Comment (fredrik-bakke) [inline src/category-theory/copresheaf-categories.lagda.md:320]: Can you add just a sentence stating that the initial copresheaf is the constant functor at the empty set?

**Review by fredrik-bakke — APPROVED** [review]

File: src/category-theory/copresheaf-categories.lagda.md (hunk 4 above)
Comment (FernandoChu) [reply src/category-theory/copresheaf-categories.lagda.md]: Done! The above one too.

**Comment by fredrik-bakke** [comment]
Oh my, it didn't display on my phone that I was a co-author on one of the commits, and it didn't show me the merge commit message before it was merged. Otherwise, I would have removed myself as coauthor. Sorry about that. Luckily this PR wasn't that big, but still.

**Pull request merged**